File: BACKEND/PickleballCourtBookingSystem/PickleballCourtBookingSystem.Core/DTOs/StatisticDto.cs
using PickleballCourtBookingSystem.Core.Entities;

namespace PickleballCourtBookingSystem.Core.DTOs;

public class StatisticDto
{
    public CourtCluster? CourtCluster { get; set; }
    public List<Booking>? Bookings { get; set; }
    public double TotalRevenue { get; set; }
    public int TotalBookings { get; set; }
}

File: AndroidApp/app/src/main/java/com/maxholmes/androidapp/data/dto/response/StatisticDto.kt
package com.maxholmes.androidapp.data.dto.response

import com.maxholmes.androidapp.data.model.Booking
import com.maxholmes.androidapp.data.model.CourtCluster


data class StatisticDto(
    var courtCluster: CourtCluster? = null,
    var bookings: List<Booking>? = null,
    var totalRevenue: Double = 0.0,
    var totalBookings: Int = 0
)
